Controllers/AddSelectedInputsToIntervention.swift

```diff
@@ -143,11 +143,11 @@ extension AddInterventionViewController: SelectedInputCellDelegate {
         return
       }
       let efficiencyWithoutTrailing = forTrailingZero(temp: efficiency)
-      cell.surfaceQuantity.text = String(format: "or".localized, efficiencyWithoutTrailing, (unit.components(separatedBy: "/")[0]))
+      cell.surfaceQuantity.text = String(format: "Soit %d %@", efficiencyWithoutTrailing, (unit.components(separatedBy: "/")[0]))
     } else {
       efficiency = Double(quantity) / surfaceArea
       let efficiencyWithoutTrailing = forTrailingZero(temp: efficiency)
-      cell.surfaceQuantity.text = String(format: "or_per_hectare".localized, efficiencyWithoutTrailing, unit)
+      cell.surfaceQuantity.text = String(format: "Soit %d %@ par hectare", efficiencyWithoutTrailing, unit)
```

Comment:
Unlocalized string.

Reply:
This will be localized in near future, since we follow android version locales